components/button/floating-buttons.md

```diff
@@ -1,24 +1,65 @@
-# d2l-floating-buttons
+# Floating Buttons
 
 Floating workflow buttons behavior can be added by using the `<d2l-floating-buttons>` custom element. When the normal position of the workflow buttons is below the bottom edge of the viewport, they will dock at the bottom edge. When the normal position becomes visible, they will undock.
 
+<!-- docs: demo name:d2l-floating-buttons autoSize:false display:block size:medium -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/button/floating-buttons.js';
+  import '@brightspace-ui/core/components/button/button.js';
+</script>
+<style>
+  .d2l-typography p {
+    margin: 1rem;
+  }
+  d2l-button { 
+    margin-left: 0.5rem;
+  }
+</style>
+<p>
+  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci repellat cum totam! Enim, sunt. Numquam voluptate, velit quisquam ipsa molestias laudantium odit reiciendis nisi corporis voluptatibus, voluptatum sunt natus, accusantium magnam consequatur fugit officiis minima voluptatem consequuntur nam, earum necessitatibus! Cupiditate ullam repellendus, eius iure voluptas at commodi consectetur, quia, adipisci possimus, ex mollitia. Labore harum error consectetur officiis aut optio, temporibus iste nobis ducimus cumque laudantium rem pariatur. Ut repudiandae id, consequuntur quasi quis pariatur autem corporis perferendis facilis eius similique voluptatibus iusto deleniti odio officia numquam tenetur excepturi, aspernatur sunt minima aut fugiat ipsam.
+</p>
+<d2l-floating-buttons min-height="100px">
+  <d2l-button primary>Primary</d2l-button>
+  <d2l-button>Secondary</d2l-button>
+</d2l-floating-buttons>
+```
+
+<!-- docs: start hidden content -->
 ![Floating Buttons](./screenshots/floating-buttons.png?raw=true)
+<!-- docs: end hidden content -->
 
+## Floating button [d2l-floatin-button]
```

Comment:
```suggestion
## Floating Button [d2l-floating-buttons]
```

Reply:
This one's a bit weird with our pattern of non-plural components since the component is actually pluralized🤔

Reply:
Very good point, it doesn't really make sense to have it as `Floating Button` but otherwise we would have a duplicate level 1/2 headers. I think I'll change it to the plural for now and see what Jeff thinks during the review